integration-tests/src/tests/client/features/access_key_nonce_for_implicit_accounts.rs

```diff
@@ -770,6 +763,12 @@ fn test_chunk_forwarding_optimization() {
             assert_eq!(&accepted_blocks[0], block.header().hash());
             assert_eq!(test.env.clients[i].chain.head().unwrap().height, block.header().height());
         }
+
+        test.process_network_messages();
+        test.env.process_shards_manager_responses(0);
+
+        let output = test.env.propagate_chunk_state_witnesses();
```

Comment:
nit: Can add a tiny comment here saying something on the lines of, we're not really testing state witness or chunk endorsement but this is necessary for block production.

Reply:
Added a comment